src/utils/connection.ts

```diff
@@ -11,8 +11,12 @@ export function handleWsDashboard(storage: IStorage) {
   return (data: any) => {
     // update network height
     const height = data.best_block_height as number;
-    storage.setCurrentHeight(height);
-    storage.unlockUtxos(height);
+    storage.getCurrentHeight().then(async (currentHeight) => {
```

Comment:
Is there a reason to use promise here?

Reply:
Its only needed to await the promises below, but they are not actually necessary.